fv/ingress_egress_test.go

```diff
@@ -220,3 +220,65 @@ var _ = Context("with initialized Felix, etcd datastore, 3 workloads", func() {
 		})
 	})
 })
+
+var _ = Context("with Typha and Felix-Typha TLS", func() {
+
+	var (
+		etcd   *containers.Container
+		felix  *infrastructure.Felix
+		client client.Interface
+		w      [3]*workload.Workload
+	)
+
+	BeforeEach(func() {
+		options := infrastructure.DefaultTopologyOptions()
+		options.WithTypha = true
+		options.WithFelixTyphaTLS = true
+		felix, etcd, client = infrastructure.StartSingleNodeEtcdTopology(options)
+
+		// Install a default profile that allows workloads with this profile to talk to each
+		// other, in the absence of any Policy.
+		defaultProfile := api.NewProfile()
+		defaultProfile.Name = "default"
+		defaultProfile.Spec.LabelsToApply = map[string]string{"default": ""}
+		defaultProfile.Spec.Egress = []api.Rule{{Action: api.Allow}}
+		defaultProfile.Spec.Ingress = []api.Rule{{
+			Action: api.Allow,
+			Source: api.EntityRule{Selector: "default == ''"},
```

Comment:
I'd be tempted to make sure there's at least one path that gets denied here, just to be 100% sure that policy got applied rather than some fail-open bug.  Finding it pretty hard to think of a mechanism for _only_ TLS mode to fail open though so I'll leave it up to you.